src/Organisms/NavElement/NavElementStyled.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { COLORS } from '../../Styles/varsStyledComponents';
+
+export const LinkWrapper = styled.div`
+  height: 100%;
+  width: 33%;
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  cursor: pointer;
+
+  .arrowLeft {
+    color: ${COLORS.grey200};
+    font-size: 30px;
+  }
+
+  .back {
+    font-family: Mulish;
+    font-size: 14px;
+    color: ${COLORS.grey200};
+  }
+`;
+
+export const NavElementContainer = styled.div`
+  margin: 0;
```

Comment:
think about left and right margin of your container (I propose 1REM on the left and 1REM on the right)